test/units/test_constants.py

```diff
@@ -196,10 +196,10 @@ def test_value_type_none(self, cfgparser):
         assert constants.get_config(cfgparser, 'defaults', 'unknown', 'ANSIBLE_TEST_VAR', None, value_type='none') is None
 
     def test_value_type_path(self, cfgparser, user, cfg_file):
-        assert constants.get_config(cfgparser, 'defaults', 'unknown', 'ANSIBLE_TEST_VAR', '~/local', value_type='path') == os.path.join(user['home'], 'local')
-        assert constants.get_config(cfgparser, 'defaults', 'unknown', 'ANSIBLE_TEST_VAR', 'local', value_type='path') == 'local'
-        assert constants.get_config(cfgparser, 'defaults', 'unknown', 'ANSIBLE_TEST_VAR', 'local', value_type='path', expand_relative_paths=True) \
-                == os.path.join(cfg_file, 'local')
+        assert constants.get_config(cfgparser, 'defaults', 'unknown', 'ANSIBLE_TEST_VAR', '~/local', value_type='path', expand_relative_paths=False) \
+                 == os.path.join(user['home'], 'local')
```

Comment:
The following indentation should pass pep8 (with the `==` starting under the `r` in `assert`):

```
        assert constants.get_config(cfgparser, 'defaults', 'unknown', 'ANSIBLE_TEST_VAR', '~/local', value_type='path', expand_relative_paths=False) \
            == os.path.join(user['home'], 'local')
```